### PR TITLE
Add group name to webhooks index in admin panel

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/YcYyBtYY/1931-1-webhooks-panel-should-show-group-name)

screen shot 2016-10-13 at 4 58 51 pm

### CODE REVIEW

**Comment by HSalmon** [comment]
@gdpelican, would you be up for taking a look at this?

File: app/admin/webhooks.rb [hunk 1]
```diff
@@ -33,6 +33,19 @@ def webhook_params
   filter :event_types
   filter :created_at
 
+  index do |webhook|
+    column :hookable_id
+    column :hookable_type
+    column 'Hookable Name' do |w|
+      if w.hookable_type == 'Group'
```
Comment (gdpelican) [inline app/admin/webhooks.rb]: This should be simply `w.hookable.name if w.hookable.respond_to?(:name)`

**Comment by HSalmon** [comment]
screen shot 2016-10-18 at 2 47 09 pm

**Comment by HSalmon** [comment]
@gdpelican, is this good to go?

**Pull request merged**